internal/controllers/aggregation/composition.go

```diff
@@ -72,6 +72,8 @@ func (c *compositionController) aggregate(synth *apiv1.Synthesizer, comp *apiv1.
 		return copy
 	}
 
+	// TODO: New status for synthesis retry backoff
```

Comment:
Will this be addressed in this PR?

Reply:
I am planning on folding the status aggregation logic into the composition controller but didn't want to make this PR any larger. So I'll follow up once this merges.